sympy/core/mul.py

```diff
@@ -423,6 +423,11 @@ def _gather(c_powers):
             changed = False
             for b, e in c_powers:
                 if e.is_zero:
+                    # canceling out infinities yields NaN
+                    if (b.is_Add or b.is_Mul) and any(infty in b.args \
```

Comment:
Could `b` itself be one of the infinities, not an Add or Mul?

Backslash is not needed for continuation here. (PEP 8: Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.)

Reply:
If an infinity is divided by infinity, or multiplied by 0, this code is not reached: the checks returning NaN in those cases are in `__mul__` and `__div__` methods of the Infinity classes, implemented in numbers.py ([Example](https://github.com/sympy/sympy/blob/master/sympy/core/numbers.py#L2753)).

I will get rid of the backslash